docs/2_2_common_schema.adoc

```diff
@@ -619,15 +619,31 @@ It provides the static information of all exposed variables and is defined as:
 
 image::images/ModelVariables_schema.png[width=100%, align="center"]
 
-The `ModelVariables` element consists of an ordered set of `ScalarVariable` elements (see figure above).
-The first element has `index = 1`,
-the second `index=2`, etc.
-This `ScalarVariable` `index` is used in element `ModelStructure` to uniquely and efficiently refer to `ScalarVariable` definitions.
-A `ScalarVariable` represents a variable of primitive type, like a real or integer variable.
-For simplicity,
-only scalar variables are supported in the schema file in this version and structured entities (like arrays or records) have to be mapped to scalars.
-The schema definition is present in a separate file `fmi3ScalarVariable.xsd`.
-The attributes of `ScalarVariable` are:
+The `ModelVariables` element consists of an ordered set of `Variable` elements (see figure above).
```

Comment:
When citing form the XML, shouldn't we keep the angle brackets (e.g. `<Variable>`) and use the plain word "variable" when we refer to the abstract idea of a variable?

Reply:
Just checked, for all other xml tags we don't use angle brackets in the schema description to refer to the tag, e.g.  we write `ModelVariables` instead of `<ModelVariables>`.
It might be a good idea to use angle brackets in the specification whenever an xml tag/element is referred.
For variables we have to consider that we should introduce a meta-tag `Variable`. For this meta-tag wie should not use the angle brackets but maybe curly brackets to not confuse it with an actual tag/element.